cvatjs/src/server-proxy.js

```diff
@@ -179,10 +179,10 @@
             }
 
             async function logout() {
-                const { backendAPI } = window.cvat.config;
+                const host = window.cvat.config.backendAPI.slice(0, -7);
```

Comment:
Why?

Reply:
@nmanovic 
backendAPI is ```host/api/v1```
authorization url is ```host/auth```